src/test/user-query.test.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from "chai";
+import request, { SuperTest, Test } from "supertest";
+import { getRepository, Repository } from "typeorm";
+
+import { User } from "../entity";
+import { ErrorMessage } from "../core/error";
+import { JWTService } from "../core/security/jwt";
+import { CreateUserInput } from "../schema/schema.types";
+import { createUserEntity, requestQuery } from "./common";
+
+describe("Tests for User", () => {
+  let agent: SuperTest<Test>;
+  let repository: Repository<User>;
+
+  const user: CreateUserInput = {
+    name: "user",
+    email: "user@taqtile.com",
+    password: "1234qwer",
+    birthDate: "01-01-2000",
+  };
+
+  before(async () => {
+    agent = request(`${process.env.BASE_URL}`);
+    repository = getRepository(User);
+  });
+
+  afterEach(() => {
+    repository.delete({});
+  });
```

Comment:
```suggestion
  afterEach(() => {
    await repository.delete({});
  });
```

Provavelmente não vai acontecer porque esse delete é rápido, mas é bom tomar cuidado com essas coisas porque pode ser que o próximo teste rode sem ele ter terminado esse delete.